modules/system/system.install

```diff
@@ -236,35 +236,48 @@ function system_requirements($phase) {
 
   // Test settings.php file writability
   if ($phase == 'runtime') {
-    // Pantheon handles permissions automatically.
+    $conf_errors = array();
+    // Allow system administrators to ignore permissions hardening for the site
+    // directory. This allows additional files in the site directory to be
+    // updated when they are managed in a version control system.
+    $skip_permissions_hardening = variable_get('skip_permissions_hardening', FALSE);
+
+    if ($skip_permissions_hardening) {
+      $error_value = t('Protection disabled');
+      // If permissions hardening is disabled, then only show a warning for a
+      // writable file, as a reminder, rather than an error.
+      $file_protection_severity = REQUIREMENT_WARNING;
+    }
+    elseif (defined('PANTHEON_ENVIRONMENT')) {
+      $file_protection_severity = REQUIREMENT_WARNING;
```

Comment:
If I understand correctly, this is only getting triggered in Pantheon if settings.php doesn't exist and keep the other checks if the site is being run out of Pantheon, right?

Reply:
So, it's doing pretty much the same than before and the new skip_permissions_hardening variable doesn't really have any effect in Pantheon

Reply:
Correct, this block prevents the `else` from running on Pantheon.

I could have moved the `$file_protection_severity` down to the block below and changed the `else` to an `elseif (!...)`, but I felt that it would be better to have fewer modified lines. Maybe that doesn't matter.